docs/tutorials/how-to-test-a-contract.md

```diff
@@ -53,12 +34,12 @@ let result = demo.verify(() => {
 });
 ```
 
-If the contract is successfully executed, the return result `result.success` is `true`. Otherwise it will throw an exception.
+If the contracts method is successfully evaluated, the returned result `result.success` will be `true`. Otherwise it will throw an exception.
```

Comment:
If the contracts method is successfully evaluated, `result.success` will be `true`.